projects/glif/index.js

```diff
@@ -7,10 +7,10 @@ module.exports = {
   methodology:
     "The GLIF Pools protocol is a liquid leasing protocol for Filecoin that requires borrowers to collateralize FIL in order to borrow for their storage providing operation. This TVL calculation adds the total amount of FIL deposited into the protocol, and the total amount of locked FIL collateral by borrowers, to arrive at TVL.",
   filecoin: {
-    tvl: async (_, height, _1, { api }) => {
+    tvl: async (timestamp, _, _1, { api }) => {
```

Comment:
you cant use timestamp like that, instead you should get the timestamp from the api object